### DISCUSSION TITLE
.py to .exe Conversion

### DISCUSSION THREAD

**Discussion by vikramsamak** [TomSchimansky/CustomTkinter#341] [Q&A]
is auto-py-to-exe issue is fixed or not ?

**Comment by plugeit** [comment]
what issue can you please elaborate

**Comment by ivancasaof** [comment]
Hi there.
You must add the following folder in "Additional Files\ Add Folder":
customtkinter/
customtkinter-4.5.10.dist-info/
darkdetect/
darkdetect-0.7.1.dist-info/

And in Advanced\ hidden-import:
distutils
distutils.version
ctypes.wintypes
json
__future__

Works for me!!. Bye
